src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1854,6 +1845,19 @@ private DetectProperties() {
     //#endregion Active Properties
 
     //#region Deprecated Properties
+    
+    @Deprecated
+    public static final EnumProperty<LogLevel> LOGGING_LEVEL_COM_SYNOPSYS_INTEGRATION = 
+            EnumProperty.newBuilder("logging.level.com.synopsys.integration", LogLevel.INFO, LogLevel.class)
+            .setInfo("Logging Level", DetectPropertyFromVersion.VERSION_5_3_0)
+            .setHelp("Equivalent to setting <i>logging.level.detect</i>.",
+                    "Refer to the description of property <i>logging.level.detect</i> for additional details.")
+            .setGroups(DetectGroup.LOGGING, DetectGroup.GLOBAL)
+            .setDeprecated(
+                    "This property is being removed. Use property logging.level.detect instead. There is no distinction between the two properties.",
```

Comment:
Wording suggestion: 
`This property has been deprecated and will be removed as of version 10.0.0. Use the equivalent logging.level.detect property instead.`